dspace-api/src/main/java/org/dspace/core/AbstractHibernateDAO.java

```diff
@@ -209,7 +209,14 @@ public Query createQuery(Context context, String query) throws SQLException {
      * retrieved an exception will be thrown,
      * so only use when the criteria state uniqueness in the database.
      * @param criteriaQuery JPA criteria
-     * @return a DAO specified by the criteria
+     * @param cacheable whether or not this query should be cacheable.
```

Comment:
the context param is missing in the javadoc

Reply:
Fixed.